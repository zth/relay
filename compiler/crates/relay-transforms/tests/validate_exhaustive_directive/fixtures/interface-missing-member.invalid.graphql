fragment InterfaceMissingMember on User {
  nameRenderable @exhaustive {
    ... on PlainUserNameRenderer { __typename }
    ... on MarkdownUserNameRenderer { __typename }
    ... on ImplementsImplementsUserNameRenderable { __typename }
  }
}
