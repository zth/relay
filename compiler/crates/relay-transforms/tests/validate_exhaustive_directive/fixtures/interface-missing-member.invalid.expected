==================================== INPUT ====================================
#expected-to-throw
fragment InterfaceMissingMember on User {
  nameRenderable @exhaustive {
    ... on PlainUserNameRenderer { __typename }
    ... on MarkdownUserNameRenderer { __typename }
    ... on ImplementsImplementsUserNameRenderable { __typename }
  }
}
==================================== ERROR ====================================
✖︎ Field 'nameRenderable' marked with @exhaustive is missing selection for interface implementations: 'ImplementsImplementsUserNameRenderableAndUserNameRenderable'.

  interface-missing-member.invalid.graphql:3:3
    2 │ fragment InterfaceMissingMember on User {
    3 │   nameRenderable @exhaustive {
      │   ^^^^^^^^^^^^^^
    4 │     ... on PlainUserNameRenderer { __typename }
